### PR TITLE
Added instructions for installing Adafruit and spidev

### PR DESCRIPTION
**Pull request by beekman**

### CODE REVIEW

**Review by licarpen — CHANGES_REQUESTED** [review]
I personally am not in favor of included vnc server info - it has nothing to do with our project and I didn't use it, as it didn't end up working for me. Ryan suggested several other methods to interface with the pi.
 If folks feel strongly otherwise, I'd like to see this language cleaned up to match that found in similar documentation.  Since the documentation is written for other developers, I'd like to see the instructions reflect an assumption that developers know how to open files, uncomment lines, etc.  Some suggestions are below as an example: 
Instead of "Optional instructions to connect to the Raspberry Pi without a display or keyboard attached" it should read "Optional instructions to utilize a remote interface (not necessary for the full functionality of this application)" 
I didn't run into any issues with display size, so do not think we should include "you'll NEED to first make a few changes..."  Maybe something like "if you encounter screen dimension issues, execute `sudo nano /boot/config.txt` from the terminal and uncomment the following lines:   ("scroll through the file" is unnecessary as it is assumed that a develop knows how to do that).  Get rid of "Exit with ctrl-x... etc," as a developer should know how to exit a terminal process.     Rephrase the "On the Raspberry Pi, type  `vncserver` and hit return" to something more formal, like "execute `vncserver`.    Rephrase "The command should respond with a number of details relating to the connection, including a VNC Server catchphrase which will help you ensure you're connecting to the right device, and a phrase "New desktop is `hostname:1` (where hostname is whatever you set earlier) followed by an ip address."  It's hard to follow.

**Review by ianandrewson — CHANGES_REQUESTED** [review]
-Line 105 needs the trailing "to" removed. I also don't see the command, but it could just be me missing something?

Looks pretty solid beyond that I think.

**Review by licarpen — APPROVED** [review]

**Review by ianandrewson — APPROVED** [review]
